ci/openshiftio-appdev-docs/src/main/java/io/openshift/appdev/documentation/builder/HttpApplication.java

```diff
@@ -22,26 +22,51 @@
 import io.vertx.ext.web.handler.RedirectAuthHandler;
 import io.vertx.ext.web.handler.StaticHandler;
 
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
 public class HttpApplication extends AbstractVerticle {
 
-   private static final String INDEX_PAGE = "index.html";
-   private static final String LAUNCHER_TEMPLATE_LATEST_URL = "https://raw.githubusercontent.com/openshiftio/launchpad-templates/v5/openshift/launchpad-template.yaml";
+    private static final String propFileLocation = "src/main/resources/application.properties";
+    private static final Properties props = loadProperties();
+
+    private static Properties loadProperties() {
+        Properties props = new Properties();
+        FileInputStream  ioStream = null;
+        try {
+            ioStream = new FileInputStream(propFileLocation);
```

Comment:
If it's in `src/main/resources`, it will end up in the application JAR and could be loaded from the classpath I think?